configure.ac

```diff
@@ -19,7 +19,6 @@ AC_SUBST(LIBTORRENT_INTERFACE_VERSION_NO)
 
 AM_INIT_AUTOMAKE
 AC_CONFIG_HEADERS(config.h)
-AM_PATH_CPPUNIT(1.9.6)
```

Comment:
I was affected by this issue: https://github.com/OpenLightingProject/ola/issues/1043